src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,11 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number >= 0) {
+            number = Math.sqrt(number);
+        } else {
+            throw new IllegalArgumentException("Expected non-negative number, got " + number);
+        }
```

Comment:
good but it's better to check first whether or not number is negative and throw exception if yes